src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -356,18 +356,8 @@ export default class DefaultDeviceController
 
     // Update device and stream
     this.chosenVideoTransformDevice = device;
-    const newMediaStream = this.activeDevices['video'].stream;
```

Comment:
I don't see any change in the device controller unit tests. Do we have any to cover this?